src/Fantomas/Program.fs

```diff
@@ -411,14 +388,40 @@ let main argv =
 
             elog $"Failed to format file: {file}{message}"
 
+        let reportProfileInfos results =
```

Comment:
You should pass the `oks` and `unchanged` from `let oks, ignored, unchanged, errored = partitionResults results` here.

Reply:
Done, as you already saw.